src/DotNetty.Codecs.ProtocolBuffers/ProtobufDecoder.cs

```diff
@@ -24,6 +24,8 @@ public ProtobufDecoder(IMessageLite protoType, ExtensionRegistry extensionRegist
             this.extensionRegistry = extensionRegistry;
         }
 
+        public override bool IsSharable => true;
```

Comment:
Same thing as protobuf 3, see
https://github.com/Azure/DotNetty/blob/dev/test/DotNetty.Codecs.ProtocolBuffers.Tests/RoundTripTests.cs#L106
The builder is bound to the message type builder, not really shareable.

Reply:
Same as above, remove the 'sealed'.